tests/system/test_enrollment/test_enrollment.py

```diff
@@ -0,0 +1,255 @@
+            new_configuration = old_agent_configuration.replace('<address>MANAGER_IP</address>',
+                                                                f"<address>{network['manager_network'][0]}</address>")
+            host_manager.modify_file_content(host='wazuh-agent1', path='/var/ossec/etc/ossec.conf',
+                                             content=new_configuration)
+            messages_with_ip = messages.replace('MANAGER_IP', f"{network['manager_network'][0]}")
+        elif 'ipv6' in configuration['ip_type']:
+            new_configuration = old_agent_configuration.replace('<address>MANAGER_IP</address>',
+                                                                f"<address>{network['manager_network'][1]}</address>")
+            host_manager.modify_file_content(host='wazuh-agent1', path='/var/ossec/etc/ossec.conf',
+                                             content=new_configuration)
+            messages_with_ip = messages.replace('MANAGER_IP', f"{network['manager_network'][1]}")
+        elif 'dns' in configuration['ip_type']:
+            new_configuration = old_agent_configuration.replace('<address>MANAGER_IP</address>',
+                                                                f"<address>wazuh-manager</address>")
+            host_manager.modify_file_content(host='wazuh-agent1', path='/var/ossec/etc/ossec.conf',
+                                             content=new_configuration)
+            if 'yes' in configuration['ipv6_enabled']:
+                if 'ipv4' in configuration['manager_network'] or 'ipv4' in configuration['agent_network']:
+                    messages_with_ip = messages.replace('MANAGER_IP', f"wazuh-manager/{network['manager_network'][0]}")
+                else:
+                    messages_with_ip = messages.replace('MANAGER_IP', f"wazuh-manager/{network['manager_network'][1]}")
+            else:
+                messages_with_ip = messages.replace('MANAGER_IP', f"wazuh-manager/{network['manager_network'][0]}")
+
+        if 'ipv4' in configuration['ip_type']:
+            messages_with_ip = messages_with_ip.replace('AGENT_IP', f"{network['agent_network'][0]}")
+        elif 'ipv6' in configuration['ip_type']:
+            messages_with_ip = messages_with_ip.replace('AGENT_IP', f"{network['agent_network'][1]}")
+        elif 'dns' in configuration['ip_type']:
+            if 'yes' in configuration['ipv6_enabled']:
+                if 'ipv4' in configuration['agent_network']:
+                    messages_with_ip = messages_with_ip.replace('AGENT_IP', f"{network['agent_network'][0]}")
+                else:
+                    messages_with_ip = messages_with_ip.replace('AGENT_IP', f"{network['agent_network'][1]}")
+            else:
+                messages_with_ip = messages_with_ip.replace('AGENT_IP', f"{network['agent_network'][0]}")
+
+    with open(manager_conf_file, 'w') as file:
+        file.write(new_manager_configuration)
```

Comment:
Use `write_file` function of the file module instead.

Reply:
Done in 861bbd0